site/_includes/partials/meta.njk

```diff
@@ -28,13 +24,15 @@
 <meta property="og:type" content="website" />
 <meta property="og:url" content="{{ currentUrl }}" />
 
-{% if hero %}
+{% set socialImage = thumbnail or hero or site.defaultSocialImage %}
```

Comment:
I think I get it now: thumbnail "wins" for meta tags, and hero "wins" for actual content pages.

I think that's not incredibly obvious but I'm happy to LGTM anyway.